### PR TITLE
[DUOS-1896][risk=no] Ensure UserProperties deprecation does not break frontend

### PR DESCRIPTION
**Pull request by connorlbark**
## Partially addresses 

https://broadworkbench.atlassian.net/browse/DUOS-1896

Basic frontend changes to ensure that https://github.com/DataBiosphere/consent/pull/1650 does not break the UI.

Checked the places user props were used and made sure it didn't break; made small changes if no nil checks. Created [new story](https://broadworkbench.atlassian.net/browse/DUOS-1936) to remove all references of outdated user props.

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/components/ApplicationDownloadLink.js [hunk 1]
```diff
@@ -157,7 +157,7 @@ export default function ApplicationDownloadLink(props) {
   const externalCollaborators = getCollaborators(darInfo, 'externalCollaborators');
   const labCollaborators = getCollaborators(darInfo, 'labCollaborators');
   const researcherProps = getPropertyValuesFromUser(researcherProfile);
-  const location = (researcherProps.city).concat(', ').concat(researcherProps.state);
+  const location = (researcherProps.city && researcherProps.state ? (researcherProps.city).concat(', ').concat(researcherProps.state) : '');
```
Comment (rushtong) [inline src/components/ApplicationDownloadLink.js]: We should remove this altogether, along with the location block on line 209

File: src/libs/utils.js [hunk 2]
```diff
@@ -804,25 +804,8 @@ export const evaluateTrueString = (boolString) => {
...
-    const piAttrEmpty = isEmpty(piName) || isEmpty(piEmail);
-    return !(isThePIFalse && havePITrue && piAttrEmpty);
-  };
-
-  const stringAttrs = pick(['displayName', 'address1', 'city', 'state', 'zipCode', 'country'])(properties);
-  const stringAttrsCompleted = every((string) => !isEmpty(string))(stringAttrs);
   const institutionPresent = !isNil(institutionId);
-  const piValid = piCheck({isThePI, havePI, piEmail, piName});
-  return piValid && stringAttrsCompleted && institutionPresent;
+  return institutionPresent;
```
Comment (rushtong) [inline src/libs/utils.js]: This can be inlined and remove `institutionPresent` as a temp var.
```
return !isNil(institutionId);
```

**Review by rushtong — CHANGES_REQUESTED** [review]
Have a look at `UserProperties` in `utils.js`. There are a lot of usages of some outdated properties. For example, `CITY` is used in `AppSummary.js`, but when it is passed to a subcomponent (`ApplicantInfo`), it is not used anywhere. It would be great if we could clean up unused constants in `UserProperties` for this PR.

**Comment by connorlbark** [comment]
> Have a look at `UserProperties` in `utils.js`. There are a lot of usages of some outdated properties. For example, `CITY` is used in `AppSummary.js`, but when it is passed to a subcomponent (`ApplicantInfo`), it is not used anywhere. It would be great if we could clean up unused constants in `UserProperties` for this PR.

ohh okay i think i misinterpreted you when we talked in standup yesterday - i can remove those references and work on updating it all!

**Comment by connorlbark** [comment]
i believe that i removed all usages of deprecated user properties EXCEPT for the usages on the dar application page; I can go ahead and do that, too, but I wasn't sure if that was in the scope of this PR.

File: src/components/ApplicationDownloadLink.js [hunk 3]
```diff
@@ -156,64 +155,21 @@ export default function ApplicationDownloadLink(props) {
...
     h(Page, {style: styles.page}, [ //Researcher Info Page
       h(View, {}, [
         h(Text, {style: styles.header}, [`${darInfo.darCode} Application`]),
         h(Text, {style: styles.subHeader}, ['Applicant Information']),
-        h(View, {style: styles.flexboxContainer}, [
-          h(SmallLabelTextComponent, {
-            label: 'NIH eRA Commons ID',
-            text: `${researcherProps.eraCommonsId}`,
-            style: {marginRight: 30}
-          }),
```
Comment (rushtong) [inline src/components/ApplicationDownloadLink.js:174]: This block is something we still need. The era commons id is actually a prop on the user now - this is relatively old code.

**Comment by rushtong** [comment]
> i believe that i removed all usages of deprecated user properties EXCEPT for the usages on the dar application page; I can go ahead and do that, too, but I wasn't sure if that was in the scope of this PR.

I'm pretty sure we've already dealt with the DAR application page in previous stories, no need to look further there.

**Review by rushtong — CHANGES_REQUESTED** [review]
One small un-delete/update and I think this is ready to go!

**Review by rushtong — APPROVED** [review]
This looks good to go 👍🏽

**Review by quazi-h — APPROVED** [review]
Changes look good to me and I did not encounter any issues with the UI when running against the consent changes in a local instance.

**Pull request merged**